src/hnsw.h

```diff
@@ -85,6 +85,9 @@
 /* Optimal ML from paper */
 #define HnswGetMl(m) (1 / log(m))
 
+/* Scaling factor for HNSW index scan estimation */
+#define HnswScanScalingFactor ((double) 0.55)
```

Comment:
I'd prefer to keep this in `hnswcostestimate`, but if you feel strongly it should be separate, let's use the prefix `HNSW_COST`.

Reply:
I don't feel strongly about that - but - I do feel it should be configurable and just wanted to simplify transition if/when it happens 😛 I will move.